### DISCUSSION TITLE
配置的pgsql无法连接，提示engine pgsql not enabled or not supported, please contact admin

### DISCUSSION THREAD

**Discussion by YuanPeter** [hhyo/Archery#2773] [Q&A]
配置好了pgsql的实例，但是无法连接。
![f7f651a1af421c4000edb20fed3bbdd](https://github.com/user-attachments/assets/5ba099d7-a50c-4d75-bc38-a6c2f208611c)

这个是需要再setting.py里面配置什么东西吗，我这边登录服务器直连都是OK的。
下图是我这边当时setting.py修改的内容，
![72d465308105b65849fe581d79c8b3e](https://github.com/user-attachments/assets/cf5fd7b5-f4b9-415b-845f-c838e7fca7df)

**Comment by LeoQuote** [comment]
建议先升最新版或master再看能否复现

**Reply by YuanPeter** [reply]
最新版 也是这样哈。

**Reply by LeoQuote** [reply]
如果确认master分支的代码也是这样，可以重新提交下issue，这次把复现的步骤描述一下，如使用的镜像tag， hash，部署方式，修改过的配置文件，部署方式等，谢谢。
